blockchain/beacon/src/drand.rs

```diff
@@ -53,6 +54,29 @@ pub struct DrandConfig<'a> {
     pub chain_info: ChainInfo<'a>,
     /// Network type
     pub network_type: DrandNetwork,
+    // Tokio Runtime config
+    pub tokio: TokioConfig,
+}
+
+#[derive(serde::Deserialize, serde::Serialize, PartialEq, Eq, Clone)]
+pub struct TokioConfig {
+    pub worker_threads_number: usize,
```

Comment:
I think it is a bit clearer to use same naming than the one used in Tokio:

so `worker_threads_number` -> `worker_threads`
so `max_number_of_blocking_threads` -> `max_blocking_threads`
etc

That's what we are doing as well for rocksdb/paritydb.

Reply:
Done